src/main/java/uk/gov/hmcts/darts/usermanagement/component/impl/UserManagementQueryImpl.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.darts.usermanagement.component.impl;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.TypedQuery;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.ParameterExpression;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity_;
+import uk.gov.hmcts.darts.usermanagement.component.UserManagementQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+@Component
+@RequiredArgsConstructor
+public class UserManagementQueryImpl implements UserManagementQuery {
+
+    private final EntityManager em;
+
+    public List<UserAccountEntity> getUser(String emailAddress) {
```

Comment:
Is this necessary over just using the JPA repository? i.e https://github.com/hmcts/darts-api/blob/master/src/main/java/uk/gov/hmcts/darts/common/repository/UserAccountRepository.java#L16C29-L16C57

Reply:
Yes as advised by Chris B this allows us to modify the query going forward as expected.

Reply:
Yes as advised by Chris B this allows us to modify the query going forward as expected.